lib/realtime/data/vehicle_position.ex

```diff
@@ -17,7 +17,7 @@ defmodule Realtime.Data.VehiclePosition do
           timestamp: DateTime.t(),
           # GTFS-RT id, e.g. "R-54826B16".
           vehicle_id: String.t() | nil,
-          trip: nil
+          trip_id: String.t() | nil
```

Comment:
After deliberating (see my [outdated comments](https://github.com/mbta/orbit/pull/323#discussion_r2121604918)) I'm opting to just store the trip ID, mapped over from the vehicle position's `TripDescriptor`, since, after checking with Eddie, it turns out that (for the red line trains serving a current trip) the trip ID is always expected to exist, and we don't need other matching criteria, at least within a given service date.

If we want to keep the unused `trip` field for future use, let me know.